src/daemon/control.rs

```diff
@@ -1060,6 +1075,8 @@ pub struct HistoryEvent {
     pub amount: Option<u64>,
     pub fee: Option<u64>,
     pub txid: Txid,
+    #[serde(serialize_with = "serialize_tostr_vec")]
```

Comment:
Why is this necessary? We've never used it for other lists of outpoints to return.